drivers/netdev_ieee802154/netdev_ieee802154.c

```diff
@@ -151,6 +151,13 @@ int netdev_ieee802154_get(netdev_ieee802154_t *dev, netopt_t opt, void *value,
             res = sizeof(l2filter_t **);
             break;
 #endif
+        case NETOPT_MAX_PACKET_SIZE:
+            assert(max_len >= sizeof(int16_t));
+            *((uint16_t *)val) = (IEEE802154_FRAME_LEN_MAX -
```

Comment:
`/home/hydrazine/dev/RIOT/drivers/netdev_ieee802154/netdev_ieee802154.c:156:27: error: 'val' undeclared (first use in this function)`

I guess this should be `value` instead ;)

Reply:
Oh oh. I fixed this right after PR, but i forgot to push it, sorry.

Reply:
No worries.

In the future please use a fixup commit instead of force pushing. Using fixup commits makes it easy for the reviewer to track changes between revisions  and they can be automatically squashed before merging.

Reply:
You are right. This was meant to go into the PR and i just missed the push, this was allready fixed before and i just pushed it now without touching it, thats why i commented it. Sorry for the inconvenience :)

Reply:
It's fine, luckily for me this PR is not that big :)